04-Multi-Band-Rasters-In-R.Rmd

```diff
@@ -11,13 +11,14 @@ category: spatio-temporal-workshop
 tags: [raster-ts-wrksp, raster]
 mainTag: raster-ts-wrksp
 description: "This lesson explores how to import and plot a multi-band raster in
-R. It also covers how to plot a three band color image plotRGB command in R"
-code1: 
+R. It also covers how to plot a three-band color image using the `plotRGB()`
```

Comment:
Do we not want any to have the () after? If so, lots of changes in all lessons.  Not changing this one until this is decided.